tests/core/test_system_project.py

```diff
@@ -0,0 +1,42 @@
+systemProjectLabel = "authz.management.cattle.io/system-project"
+defaultProjectLabel = "authz.management.cattle.io/default-project"
+initial_system_namespaces = set(["kube-system",
+                                "cattle-system",
+                                 "kube-public"])
+
+
+def test_system_project_created(admin_cc):
+    projects = admin_cc.management.client.list_project(
+               clusterId=admin_cc.cluster.id)
+    initial_projects = {}
+    initial_projects["Default"] = defaultProjectLabel
+    initial_projects["System"] = systemProjectLabel
+    required_projects = []
+
+    for project in projects:
+        name = project.data_dict()['name']
+        if name in initial_projects:
+            projectLabel = initial_projects[name]
+            assert project.data_dict()['labels'].\
+                data_dict()[projectLabel] == 'true'
+            required_projects.append(name)
+
+    assert len(required_projects) == len(initial_projects)
+
+
+def test_system_namespaces_assigned(admin_cc, admin_pc):
+    pc = admin_pc
```

Comment:
you should not need admin_pc. You are using it to get the cluster client, which is at admin_cc.client
Also, yay tests!